rites/roman1969/src/locales/pt-br.ts

```diff
@@ -276,6 +290,7 @@ export const locale: Locale = {
       'São Dionísio, bispo e mártir, patrono da Cidade e da Diocese de Saint-Denis',
     divine_mercy_sunday: '2º Domingo do Tempo Pascal ou Domingo da Misericórdia',
     dominic_de_guzman_priest: 'São Domingos, presbítero',
+    dulce_lopes_pontes_virgin: 'Santa Dulce Lopes Pontes, virgem e religiosa',
```

Comment:
Provide a source for the Portuguese celebration names please. Each celebration must have a single source, ideally primary source (e.g. Portuguese Roman Missal or a decree issued by the Holy See or AAS or similar) or sendary source (e.g. published by an episcopal conference or its liturgical commission) or at least a tertiary sources (anything else, like Wikipedia).